src/modules/Update.js

```diff
@@ -0,0 +1,34 @@
+const listItems = document.getElementById('list-container');
+
+export default function editTask(element) {
```

Comment:
When I try to edit a task, I am presented with the following error:

<img width="541" alt="image" src="https://github.com/AleWaweru/TODO-LIST/assets/37116322/1c543056-52b9-4730-9e2c-2bab3056acab">

Please double check your edit function and make sure it works as expected without producing any errors🙏

Reply:
This error is not showing on my end, can we have a call please because I do not understand changes you are requesting.Thanks.

Reply:
done